spec.html

```diff
@@ -20773,6 +21645,21 @@ <h1>Runtime Semantics: Evaluation</h1>
         1. Set _value_.[[SourceText]] to the source text matched by |ClassExpression|.
         1. Return _value_.
       </emu-alg>
+      <emu-grammar>ClassElementName : PrivateIdentifier</emu-grammar>
+      <emu-alg>
+        1. Let _privateIdentifier_ be StringValue of |PrivateIdentifier|.
+        1. Let _privateEnvRec_ be the running execution context's PrivateEnvironment.
+        1. Assert: _privateEnvRec_ has a binding for _privateIdentifier_.
+        1. If the binding for _privateIdentifier_ in _privateEnvRec_ is an uninitialized binding, then
+          1. Let _privateName_ be NewPrivateName(_privateIdentifier_).
+          1. Perform ! _privateEnvRec_.InitializeBinding(_privateIdentifier_, _privateName_).
```

Comment:
Currently, the second arg to `InitializeBinding` is supposed to be a language value, so you'll have to change some stuff to allow it to be a Private Name.